src/screens/VisualTests/BuildResults.tsx

```diff
@@ -99,6 +122,50 @@ export const BuildResults = ({
     />
   );
 
+  // If there are no tests yet, there is no baseline for this story. User needs to create one.
+  const isCompletelyNewStory = "testsForStory" in selectedBuild && storyTests.length === 0;
+
+  if (isCompletelyNewStory) {
+    return (
+      <Sections>
+        <Section grow>
+          <Container>
+            <Heading>New story found</Heading>
+            <CenterText>
+              Take an image snapshot of this story to save its “last known good state” as a test
+              baseline. This unlocks visual regression testing so you can see exactly what has
+              changed down to the pixel.
+            </CenterText>
+            {localBuildProgress ? (
+              <CenterText style={{ display: "flex", flexDirection: "column", gap: 10, width: 200 }}>
+                <ProgressTrack>
+                  {typeof localBuildProgress.buildProgressPercentage === "number" && (
```

Comment:
Can't we just check for the existence of `localBuildProgress`? What happens when:

a) you run a build
b) it finishes
c) you create a new story?

Do you end up seeing a "full" progress bar? We might want to check that the local build is also a different build to the selected build?

(Also can we make sure we have stories for these edge cases)

Reply:
This is copied from the NoBuild screen. I extracted it to its own component and added stories similar to the BuildProgressLabel. The additional check that it is number was needed. For some reason, initialized story showed an unexpected "0" there.

Testing the case when it finishes and you create a new story, but I assume it would show the new story ui since the story wouldn't be found on selectedBuild.
I found the issue you were thinking of and fixed it. 

I'm not s